docs/src/snap/howto/proxy.md

```diff
@@ -7,6 +7,9 @@ network environment, such access is usually controlled through proxies.
 On Ubuntu and other Linux operating systems, proxies are configured through
 system-wide environment variables defined in the `/etc/environment` file.
 
+To set up a proxy using squid follow the
+[how-to-install-a-squid-server][squid] tutorial.
+
 ## Adding proxy configuration for the k8s snap
 
 Edit the `/etc/environment` file and add the relevant URLs
```

Comment:
Edit this and point people to create a `/etc/systemd/system/snap.k8s.containerd.d/10-env.conf` file instead